### ISSUE TITLE
TomTom right click menu causes extreme lag with Questie

### ISSUE THREAD

**Issue by AeroScripts** [AeroScripts/QuestieDev#880]
If both questie and TomTom are installed, right clicking on the arrow or a note on the map causes extreme lag the next time the map is opened (5-6 seconds of delay)

This happens with just TomTom, but the lag is only about 500-800ms when opening the map, so questie brings the problem to an unplayable level

Not sure what's going on

**Comment by AeroScripts** [comment]
While developing the tracker right click menu I was able to reproduce this issue but something very strange happened. When opening the right click menu many times it slowly became lagged like with the TomTom right click menu. However, when opening the world map, the tracker right click menu would get filled with every zone name, then the game would freeze, then the map opens. Seems to be related to UIDropDownMenu again

So I switched the tracker right-click menu to use this: https://www.wowace.com/projects/libuidropdownmenu
Using L_UIDropDownMenu instead of UIDropDownMenu solved the problem
